minigalaxy/ui/window.py

```diff
@@ -66,6 +66,10 @@ def __init__(self, name="Minigalaxy"):
         if not os.path.exists(THUMBNAIL_DIR):
             os.makedirs(THUMBNAIL_DIR, mode=0o755)
 
+        # Create the covers directory
+        if not os.path.exists(COVER_DIR):
```

Comment:
Could you move the creation of empty directories for thumbnails and covers to a ``make_directories`` function? Flake8 is starting to find this function to complex now.

Reply:
Fixed.